.github/workflows/checks.yml

```diff
@@ -0,0 +1,27 @@
+name: checks
+
+on:
+  pull_request:
+  push:
+    branches: [master]
+
+env:
+  CI: true
+
```

Comment:
```suggestion
```

No need, it's there by [default](https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables).